docs/dashboard.md

```diff
@@ -97,3 +97,70 @@ index=_internal source=*<addon_name>* ERROR
 ```
 
 > Note: <addon_name> is being replaced by the actual value during the build time.
+
+<br>
+# Custom components
+
+UCC also supports adding your own components to the dashboard. To do this, create a **dashboard_components.txt** file in the addon's base directory. 
```

Comment:
```suggestion
UCC also supports adding your own components to the dashboard. To do this, create a **dashboard_components.txt** file in the add-on's root directory. 
```

Reply:
Let's also add `globalConfig.json` to the proposed directory structure so it is more clear to people.